File: code-coaches/easy/vowel-counter/vowel_counter.cpp
#include <iostream>
#include <string>
#include <algorithm>

int getVowelsCount(const std::string& sentence)
{
    std::string vowels = "aeiou";
    int count = 0;
    for (char c : sentence)
    {
        if (vowels.find(std::tolower(c)) != std::string::npos)
            count++;
    }
    return count;
}

int main()
{
    std::string sentence;
    std::getline(std::cin, sentence);

    std::cout << getVowelsCount(sentence) << std::endl;

    return 0;
}

File: code-coaches/easy/vowel-counter/VowelCounter.cs
using System;

public class VowelCounter
{
    static int GetVowelsCount(string sentence)
    {
        string vowels = "aeiou";
        int count = 0;
        foreach (char c in sentence.ToLower())
        {
            if (vowels.Contains(c))
                count++;
        }

        return count;
    }

    static void Main(string[] args)
    {
        string sentence = Console.ReadLine();
        Console.WriteLine(GetVowelsCount(sentence));
    }
}
